doc/api/events.md

```diff
@@ -935,13 +935,415 @@ if the `EventEmitter` emits `'error'`. It removes all listeners when
+the Node.js environment.
+
+### Node.js `EventTarget` vs. DOM `EventTarget`
+
+There are four key differences between the Node.js `EventTarget` and the
+[`EventTarget` Web API][]:
+
+1. There is no concept of a hierarchical DOM and event propagation in Node.js.
+   That is, an event dispatched to an `EventTarget` does not propagate through
+   a hierarchy of nested target objects that may each have their own set of
+   handlers for the event.
+2. In the Node.js `EventTarget`, any object with a `type` property whose value
+   is a string may be dispatched as an `event`.
+3. In the Node.js `EventTarget`, if an event listener is an async function
+   or returns a `Promise`, and returned `Promise` rejects, the rejection will
+   be automatically captured and handled the same way as a listener that throws
+   synchronously (see [`EventTarget` Error Handling][] for details).
+4. The Node.js `EventTarget` implements an additional set of APIs that allow it
+   to closely emulate a subset of the `EventEmitter` API.
+
+### `EventTarget` vs. `EventEmitter`
+
+The Node.js `EventTarget` object implements a modified subset of the
+`EventEmitter` API that allows it to closely emulate an `EventEmitter` in
+certain situations. It is important to understand, however, that an
+`EventTarget` is *not* an instance of `EventEmitter` and cannot be used in
+place of an `EventEmitter` in most cases.
+
+1. Unlike `EventEmitter`, any given `listener` can be registered at most once
+   per event `type`. Attempts to register a `listener` multiple times will be
+   ignored.
+2. While the Node.js `EventTarget` exposes an `emit()` function that uses the
+   same signature as `EventEmitter`, the event will be dispatched to registered
+   listeners as a single event object composed as an array with an additional
+   `type` property rather than as separate arguments.
+3. The Node.js `EventTarget` does not emulate the full `EventEmitter` API.
+   Specifically the `prependListener()`, `prependOnceListener()`,
+   `rawListeners()`, `setMaxListeners()`, `getMaxListeners()`, and
+   `errorMonitor` APIs are not emulated.
+4. The Node.js `EventTarget` does not implement any special default behavior
+   for events with type `'error'`.
+5. The Node.js `EventTarget` supports `EventListener` objects as well as
+   functions as handlers for all event types.
+
+### Event Listener
+
+Event listeners registered for an event `type` may either be JavaScript
+functions or objects with a `handleEvent` property whose value is a function.
+
+In either case, the handler function will be invoked with the `event` argument
+passed to the `eventTarget.dispatchEvent()` function.
+
+Async functions may be used as event listeners. If an async handler function
+rejects, the rejection will be captured and be will handled as described in
+[`EventTarget` Error Handling][].
+
+An error thrown by one handler function will not prevent the other handlers
```

Comment:
I would expect an error thrown by one handler to reach the host error tracker and cause the process to (usually) exit.

Reply:
Well, it propagates to process.on('uncaughtException'). Whether or not that's the best here remains to be determined.

Reply:
I think that's required by the html spec - but I'm not sure that's intentional since the browser's error handling model is quite different and I don't think users would normally expect that to terminate.

As a data point deno's just throws (though they have all the phase mumbo-jumbo) and afaik no one complained. Then again Deno doesn't have many users yet so there's that.

Reply:
We could easily emit `process.on('error')` here instead.

Reply:
Updated to emit `process.on('error')` instead.

Reply:
@jasnell wait, I'm not sure that's better, does `error` have any behavior.